coords_scraper.py

```diff
@@ -0,0 +1,41 @@
+from typing import Tuple
+
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+from lxml import etree
+
+
+def get_school_coord(rspoValue: int) -> Tuple[float, float]:
+    """Scrap school coords from https://rspo.gov.pl/rspo/{rspoValue}
+
+    Args:
+        rspo (int): The rspo value
+
+    Returns:
+        Tuple[float, float]: School coords (LAT, LNG)
+    """
+
+    response = requests.get(f"https://rspo.gov.pl/rspo/{rspoValue}", verify=False)
+    soup = BeautifulSoup(response.text, "html.parser")
+    dom = etree.HTML(str(soup))
+    location = dom.xpath(
+        "/html/body/div[1]/section/div/div/div[1]/div[2]/div/div[2]/div[9]/script"
```

Comment:
Takie wybieranie po ścieżce jest dość podatne na błędy przy ew. zmianach na stronie (pomijam, że średnio można się domyślić, czego właściwie szukamy). 

```suggestion
    MAP_POINT_PATTERN = re.compile(".*var map_point =  (.*?);.*")
    location = soup.find("script", text=MAP_POINT_PATTERN)
```